mkt/asset_bundles.py

```diff
@@ -221,6 +221,7 @@
     ),
     'mkt/reviewers': (
         'js/lib/moment-with-langs.min.js',  # JS date lib.
+        'js/common/buckets.js',
```

Comment:
This is temporary, to test that the build does fail if `compress_assets` returns an error.